File: Project2_Java/src/pl/pg/oop/project2/organisms/plants/Belladonna.java
package pl.pg.oop.project2.organisms.plants;

import pl.pg.oop.project2.organisms.OrganismsEnum;
import pl.pg.oop.project2.organisms.World;

import java.awt.*;

public class Belladonna extends DangerousPlant {
    private final static int DEFAULT_STRENGTH = 99;
    public Belladonna(World world, Point position) {
        super(world, position);
        this.TypeOfOrganism = OrganismsEnum.BELLADONNA;
        this.strength = DEFAULT_STRENGTH;
    }

    @Override
    public String toString() {
        return "Belladonna";
    }
}

File: Project3_Python/Plants/Belladonna.py
import World
from Dangerous_Plant import Dangerous_Plant
from Organism import OrganismsEnum


class Belladonna(Dangerous_Plant):
    __DEFAULT_STRENGTH = 99

    def __init__(self, world, position):
        super().__init__(world, position)
        self.type_of_organism = OrganismsEnum.Belladonna
        self.strength = Belladonna.__DEFAULT_STRENGTH

    def __str__(self):
        return "Belladonna"
